eng/Version.Details.xml

```diff
@@ -16,6 +16,14 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>28d9452d7e2ae4e98a1df735b90b03d3cac1f4e7</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="6.0.0-preview.4.21211.7">
```

Comment:
I would move these two and XUnit under toolset - they are not really product impacting dependencies, right?

Reply:
I am also trying to remember if these are good packages to depend on as their version becomes stable near the time of releasing a runtime (say, 6.0), but their blob version is unstable. It might be better to depend from something that's always unstable (say, the VS ingestion artifacts).

Reply:
So you think I should be using something like VS.Redist.Common.NetCore.SharedFramework.x64.6.0 instead of Microsoft.NETCore.App.Runtime.win-x64?

Reply:
Fixed

Reply:
Yes, and an equivalent one for ASP

Reply:
Fixed (again) :-)